packages/core/drizzle/0091_bigint_evaluations_result_config.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "latitude"."evaluation_configuration_numerical" ALTER COLUMN "min_value" SET DATA TYPE bigint;--> statement-breakpoint
+ALTER TABLE "latitude"."evaluation_configuration_numerical" ALTER COLUMN "max_value" SET DATA TYPE bigint;
```

Comment:
what types were they before?

Reply:
"numeric", which for some reason is stored as a string. Anyways, this has to match with the value of `EvaluationResultableNumber`, which is a `bigint`